### PR TITLE
HEEDLS-486 Add preview button and refactor into partials

### PR DESCRIPTION
**Pull request by AlexJacksonDS**
Added a preview button link that links to the old site and refactored the Centre configuration into partials

Ran all unit tests and checked mobile layout.

image

image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/_CentreConfigurationCentreContentDetails.cshtml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CentreConfiguration
+@using Microsoft.Extensions.Configuration
+@model CentreConfigurationViewModel
```
Comment (ibrahimmunir14) [inline DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/_CentreConfigurationCentreContentDetails.cshtml:4]: Do we need lines 1 and 3 here? Same for the ManagerDetails vc.

**Review by ibrahimmunir14 — CHANGES_REQUESTED** [review]
All looks good to me, except for the configuration injection in two of the vcs.

File: DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/_CentreConfigurationCentreContentDetails.cshtml (hunk 1 above)
Comment (AlexJacksonDS) [reply DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/_CentreConfigurationCentreContentDetails.cshtml:4]: Ah no I don't think we do, just copied and pasted them and the code cleanup didn't get rid of them again

**Review by ibrahimmunir14 — APPROVED** [review]

**Review by stellake — APPROVED** [review]
Looks good - do you mind telling David to update his branch once this is merged as he's working on Edit centre details page? 🙏

**Pull request merged**